test/test_collector.py

```diff
@@ -989,6 +989,83 @@ def test_collector_output_keys(collector_class, init_random_frames, explicit_spe
     del collector
 
 
+@pytest.mark.parametrize("device", ["cuda", "cpu"])
```

Comment:
You should probably use ˋget_available_devices` here as some tests run on machines that don't have a cuda device